ntoskrnl/ke/mp.c

```diff
@@ -34,35 +32,27 @@ KeStartAllProcessors()
      /* Prep some globals for the AP */
     PVOID KernelStack;
     PVOID DPCStack;
-    PVOID PAPInfo;
+    PAPINFO APInfo;
     KPROCESSOR_STATE ProcessorState;
     KDESCRIPTOR GdtDesc, IdtDesc;
 
-    HighestPhysicalAddress.QuadPart = UINT64_MAX;
     __sgdt(&GdtDesc.Limit);
     __sidt(&IdtDesc.Limit);
 
     /* Attempt to allocate memory for the new setup */
-    SIZE_T APInfoSize = sizeof(struct APInfo);
+    SIZE_T APInfoSize = sizeof(APInfo);
```

Comment:
This gives you the size of APInfo, which is a pointer, so the result is 4/8. What you want is `sizeof(*APInfo)` or `sizeof(APINFO)`.